app/Http/Controllers/CustomerController.php

```diff
@@ -2,9 +2,17 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\CreateCustomerRequest;
+use App\Jobs\CreateCustomerJob;
+use Illuminate\Http\RedirectResponse;
 
 class CustomerController extends Controller
 {
-    //
+    public function store(CreateCustomerRequest $request): RedirectResponse
```

Comment:
every thing is wrong about this method, this task should not even be a controller , it should be a job, that would be registered to tags, and the code is not good either, and even if this was a controller task, no swagger documentation